python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_functional_chaos_graphs.py

```diff
@@ -50,11 +50,11 @@ def drawFamily(factory, degreeMax=5):
 # Draw the 5-th first members of the Jacobi family.
 
 # %%
-# Create the Jacobi polynomials family using the default Jacobi.ANALYSIS
+# Create the Jacobi polynomials family using the default analysis  
 # parameter set
 alpha = 0.5
 beta = 1.5
-jacobiFamily = ot.JacobiFactory(alpha, beta)
+jacobiFamily = ot.JacobiFactory(alpha, beta, ot.JacobiFactory.ANALYSIS)
```

Comment:
If the default parametrization is ot.JacobiFactory.PROBABILITY, then we must explain that we use ot.JacobiFactory.ANALYSIS to specifically use that parametrization. If the default parametrization is ot.JacobiFactory.ANALYSIS, therefore setting this option has no interest. I suggest use both parametrizations (e.g. with a loop over the two parametrizations) to show the difference in the plot. This way, the meaning of this parameter will be clear and the feature will have an comprehensive example.